cli/command/load_balancer.go

```diff
@@ -10,6 +18,497 @@ func NewLoadBalancerCommand(b *CommandBase) *LoadBalancerCommand {
+		interval, err := strconv.Atoi(interval)
+		if err != nil {
+			return err
+		}
+
+		requiresUpdate = true
+		healthCheck.Interval = interval
+	}
+
+	if timeout := c.String("healthcheck-timeout"); timeout != "" {
+		timeout, err := strconv.Atoi(timeout)
+		if err != nil {
+			return err
+		}
+
+		requiresUpdate = true
+		healthCheck.Timeout = timeout
+	}
+
+	if healthyThreshold := c.String("healthy-threshold"); healthyThreshold != "" {
+		healthyThreshold, err := strconv.Atoi(healthyThreshold)
+		if err != nil {
+			return err
+		}
+
+		requiresUpdate = true
+		healthCheck.HealthyThreshold = healthyThreshold
+	}
+
+	if unhealthyThreshold := c.String("unhealthy-threshold"); unhealthyThreshold != "" {
+		unhealthyThreshold, err := strconv.Atoi(unhealthyThreshold)
+		if err != nil {
+			return err
+		}
+
+		requiresUpdate = true
+		healthCheck.UnhealthyThreshold = unhealthyThreshold
+	}
+
+	loadBalancerID, err := l.resolveSingleEntityIDHelper("load_balancer", args["LOADBALANCER_NAME"])
+	if err != nil {
+		return err
+	}
+
+	loadBalancer, err := l.client.ReadLoadBalancer(loadBalancerID)
+	if err != nil {
+		return err
+	}
+
+	if requiresUpdate {
+		if healthCheck.Target != "" {
+			loadBalancer.HealthCheck.Target = healthCheck.Target
+		}
+
+		if healthCheck.Interval != 0 {
+			loadBalancer.HealthCheck.Interval = healthCheck.Interval
+		}
+
+		if healthCheck.Timeout != 0 {
+			loadBalancer.HealthCheck.Timeout = healthCheck.Timeout
+		}
+
+		if healthCheck.HealthyThreshold != 0 {
+			loadBalancer.HealthCheck.HealthyThreshold = healthCheck.HealthyThreshold
+		}
+
+		if healthCheck.UnhealthyThreshold != 0 {
+			loadBalancer.HealthCheck.UnhealthyThreshold = healthCheck.UnhealthyThreshold
+		}
+
+		req := models.UpdateLoadBalancerRequest{
+			LoadBalancerID: loadBalancerID,
+			Ports:          &loadBalancer.Ports,
+			HealthCheck:    &loadBalancer.HealthCheck,
+		}
+
+		jobID, err := l.client.UpdateLoadBalancer(req)
+		if err != nil {
+			return err
+		}
+
+		onCompleteFn := func(loadBalancerID string) error {
+			loadBalancer, err := l.client.ReadLoadBalancer(loadBalancerID)
+			if err != nil {
+				return err
+			}
+
+			l.printer.StopSpinner()
+			return l.printer.PrintLoadBalancerHealthCheck(loadBalancer)
+		}
+
+		return l.waitOnJobHelper(c, jobID, "Updating health check on load balancer", onCompleteFn)
+
+	} else {
+		l.printer.Printf("No health check update required. Displaying load balancer health check.\n")
+		return l.printer.PrintLoadBalancerHealthCheck(loadBalancer)
+	}
+}
+
+func (l *LoadBalancerCommand) list(c *cli.Context) error {
+	loadBalancerSummaries, err := l.client.ListLoadBalancers()
+	if err != nil {
+		return err
+	}
+
+	return l.printer.PrintLoadBalancerSummaries(loadBalancerSummaries...)
+}
+
+func (l *LoadBalancerCommand) read(c *cli.Context) error {
+	args, err := extractArgs(c.Args(), "LOADBALANCER_NAME")
+	if err != nil {
+		return err
+	}
+
+	loadBalancer, err := l.client.ReadLoadBalancer(args["LOADBALANCER_NAME"])
```

Comment:
This needs to follow the proper pattern for `read`. See [tlake's pr](https://github.com/quintilesims/layer0/pull/386/files#diff-9e6f000570872e0a4ac06e2e2e4ba195R177)

Reply:
:white_check_mark: